src/main/java/manufacturing/db/Storage.java

```diff
@@ -2,20 +2,48 @@
 
 import java.util.HashMap;
 import java.util.Map;
+import manufacturing.model.Car;
+import manufacturing.model.Driver;
 import manufacturing.model.Manufacturer;
 
 public class Storage {
     public static Long manufacturerId = 0L;
+    public static Long carId = 0L;
+    public static Long driverId = 0L;
```

Comment:
I like your idea with private storage but I can't get why are you making id public since every object should have methods to get their unique id - this field is more for inner operations isn't it?

Reply:
IDs were accessed in the DAO implementations in the past, and I forgot to change access modifiers after moving the save() logic to Storage